docs/Troubleshooting/Troubleshooting.md

```diff
@@ -65,6 +65,28 @@ When running Pantheon from the Docker image a number of [command line options ar
 Not including a sufficient milestone in the genesis file can lead to unexpected and inconsistent behaviour without
 specific errors. Ensure a sufficient milestone is included in the genesis file (for example, `constantinoplefixblock`). 
 
+## Illegal Reflective Access Error on Startup
+
+The following error message may be displayed on startup and does not affect the operation of Pantheon: 
```

Comment:
I would add in that this occurs with Java versions 9 or later, and that this error will not be seen with Java 8.

Reply:
Fixed